### DISCUSSION TITLE
Group Policy objects

### DISCUSSION THREAD

**Discussion by Gijsreyn** [microsoft/winget-cli#5079] [Q&A] [answered]
When I checked out the Group Policies on my Windows 10 machine, I couldn't find them under Computer Configuration -> Administrative Templates -> Windows Components -> Desktop App Installer. 
Switching to my Windows 11 machine running on Hyper-V, I could see they were baked in. 

That led me to search for some documentation in this repository and on the internet. First, I stumbled on the excellent post at: https://techcommunity.microsoft.com/blog/windows-itpro-blog/manage-windows-package-manager-with-group-policy/2346322, but the URL it provides contains the old group policy objects. I could not find any documentation in this repository that describes the installation / upgrading of Group Policy templates onto domain controllers if needed. 

Does anyone know where to find this?

**Comment by florelis** [comment, marked as answer]
You can find them in the [releases](https://github.com/microsoft/winget-cli/releases) page with the name `DesktopAppInstallerPolicies.zip`. For example, here is a link for the [policies for v1.10.30-preview](https://github.com/microsoft/winget-cli/releases/download/v1.10.30-preview/DesktopAppInstallerPolicies.zip).

@denelon, this looks like something where the docs need to improve

**Reply by denelon** [reply]
* https://github.com/microsoft/winget-cli/issues/5090
